diff --git a/node_modules/typescript/lib/typescript.d.ts b/node_modules/typescript/lib/typescript.d.ts
index 4902d45..cabc457 100644
--- a/node_modules/typescript/lib/typescript.d.ts
+++ b/node_modules/typescript/lib/typescript.d.ts
@@ -566,6 +566,7 @@ declare namespace ts {
         readonly decorators?: NodeArray<Decorator>;
         readonly modifiers?: ModifiersArray;
         readonly parent: Node;
+        readonly type?: TypeNode
     }
     export interface JSDocContainer {
     }
@@ -2558,6 +2559,10 @@ declare namespace ts {
         pattern?: DestructuringPattern;
         aliasSymbol?: Symbol;
         aliasTypeArguments?: readonly Type[];
+        types?: Type[],
+        resolvedTypeArguments?: Type[],
+        value?: unknown,
+        node?: Node
     }
     export interface LiteralType extends Type {
         value: string | number | PseudoBigInt;
